priv/templates/extended_bin

```diff
@@ -150,15 +150,15 @@ relx_get_nodename() {
     if [ -z "$COOKIE" ]; then
         "$BINDIR/erl" -boot "$REL_DIR"/start_clean \
                       -mode interactive \
-                      -boot_var ERTS_LIB_DIR "$ERTS_LIB_DIR" \
+                      -boot_var SYSTEM_LIB_DIR "$SYSTEM_LIB_DIR" \
```

Comment:
heh, I think windows extended scripts never even have a concept of that variable to begin with, seems nothing is to change on that one.

Reply:
Yea, it only has `RELEASE_DIR` I don't yet understand why it uses that when using `make_script`.